File: dotnet/DesignPatterns/CreationalDesignPatterns/AbstractFactoryDesignPattern/AbstractFactoryDesignPattern.Example1/Models/Concrete/LinuxMenu.cs
using AbstractFactoryDesignPattern.Example1.Models.Abstract;

namespace AbstractFactoryDesignPattern.Example1.Models.Concrete;

public class LinuxMenu : Menu
{
	public override void Render()
	{
		Console.WriteLine("Rendering Linux Menu");
	}
	public override void AddItem(string item)
	{
		Console.WriteLine($"Adding '{item}' to Linux Menu");
	}
}

File: java/DesignPatterns/CreationalDesignPatterns/AbstractFactoryDesignPattern/AbstractFactoryDesignPattern.Example1/src/main/java/creational/abstractfactory/example1/models/concretes/LinuxMenu.java
package creational.abstractfactory.example1.models.concretes;

import creational.abstractfactory.example1.models.abstracts.Menu;

public class LinuxMenu extends Menu {
    public void render()           { System.out.println("Rendering Linux Menu"); }
    public void addItem(String item) { System.out.println("Adding '" + item + "' to Linux Menu"); }
}
